variables.tfe.tf

```diff
@@ -0,0 +1,83 @@
+variable "tfe" {
```

Comment:
Where is the implementation for the new options on this variable?

- assessments_enabled
- auto_apply
- global_remote_state
- queue_all_runs
- speculative_enabled
- terraform_version
- trigger_prefixes
- trigger_patterns
- working_directory